src/main/java/core/basesyntax/sevice/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.sevice.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void writeToFile(String fileName, String data) {
+        try {
+            Files.write(Paths.get("src/main/resources/" + fileName + ".csv"), data.getBytes());

```

Comment:
Avoid using a hardcoded path to the resources folder. Instead, pass the file path as a parameter to the method. This will make the code more flexible and easier to maintain.